src/main/java/uk/gov/hmcts/reform/sendletter/data/LetterRepository.java

```diff
@@ -47,6 +49,19 @@ public void save(WithServiceNameAndId<Letter> letterWithServiceNameAndId, Instan
         log.info("Successfully saved letter data into database with id : {} and messageId :{}", letterWithServiceNameAndId.id, messageId);
     }
 
+    /**
+     * Updates the `sent_to_print_at` column on letter(s) with given id
+     * @return number of updated rows.
+     */
+    public int update(UUID id, LetterSentToPrintAtPatch patch) {
```

Comment:
I'd replace the patch with the date. It would be better for the repository not to know this class.

Reply:
good point